backend/tests/health.test.ts

```diff
@@ -1,17 +1,10 @@
-import { afterAll, beforeAll, describe, expect, it, jest } from '@jest/globals';
+import { afterAll, beforeAll, describe, expect, it } from '@jest/globals';
 import request from 'supertest';
 import { Server } from 'http';
 
-jest.mock('../helpers/config', () => ({
```

Comment:
This wasn't actually mocking the values and the test was passing because we were setting the `default` value to `opentok`. other than that, the key/secret pairs do not `throw` anywhere so them being empty strings did not cause any issues for this test

Reply:
yep we have to use the experimental module mock for the typescript server